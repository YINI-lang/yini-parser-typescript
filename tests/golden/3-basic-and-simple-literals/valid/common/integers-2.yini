^ Numbers
int5 = 100000             // Hundred thousand
int6 = -987654            // Negative, six digits
int7 = 2147483647         // Max 32-bit signed int
int8 = -2147483648        // Min 32-bit signed int
