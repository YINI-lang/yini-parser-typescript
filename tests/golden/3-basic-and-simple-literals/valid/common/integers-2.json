{
    "Numbers": {
        "int5": 100000,
        "int6": -987654,
        "int7": 2147483647,
        "int8": -2147483648
    }
}
